examples/step-18/step-18.cc

```diff
@@ -674,7 +674,7 @@ namespace Step18
                              const double present_timestep)
     :
     Function<dim> (dim),
-    velocity (.1),
+    velocity (.08),
```

Comment:
It was. Reducing the axial displacement prevents self-intersection occurring in the second test case. This was the result with `velocity = 0.1`:
![result](https://cloud.githubusercontent.com/assets/10460610/21867308/ece1cea4-d84d-11e6-98c0-098e683a7c00.png)
Instead, the final solution is right at the limit of self-intersection. I thought that this would be better because the result remains physically/intuitively correct.